src/IRCNetwork.cpp

```diff
@@ -1223,6 +1227,9 @@ void CIRCNetwork::SetIRCSocket(CIRCSock* pIRCSock) {
 }
 
 void CIRCNetwork::IRCConnected() {
+	for (CClient* pClient : m_vClients) {
+		pClient->IRCConnected();
```

Comment:
Does this mean that `away-notify` will be announced for clients before capability negotiation between ZNC and server?
If server happens to not support it, client will already know that it doesn't need to send WHO every second, will rely on non-working away-notify offered by ZNC, and won't know away status of anyone at all.